packages/wallets/src/metamask/pages/elements/networkList.element.ts

```diff
@@ -40,4 +59,82 @@ export class NetworkList {
       await this.networkItemBtn.getByText(chainName).click();
     });
   }
+
+  async openModalNetworkEdit(chainId: any) {
+    await this.dialogSection
+      .getByTestId(`network-list-item-options-button-0x${chainId.toString(16)}`)
+      .click();
+    await this.editNetworkButton.click();
+  }
+
+  async isNetworkExist(
+    networkName: string,
+    rpcUrl: string,
+    chainId: number,
+  ): Promise<boolean> {
+    const existNetworkByName = this.dialogSection.getByTestId(networkName);
+    if (await existNetworkByName.isHidden()) {
+      return false;
+    }
+    // that locator exists only for added networks
+    const elements = this.page.getByTestId(
+      `network-rpc-name-button-0x${chainId.toString(16)}`,
+    );
+    const rpcUrlsFound = await elements.filter({ hasText: rpcUrl }).count();
+    return rpcUrlsFound != 0;
```

Comment:
i think you should to return code
```
    if (rpcUrlsFound == 0) return false;
    return true;
```

Because rpcUrlsFound == 0 couldnt be last check for this method

Reply:
What is the different?
The `rpcUrlsFound` can be only `Number` (the `count()` doesn't return another type), so i think my realisation is correct

Reply:
reverted)